<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Register</class>
 <widget class="QMainWindow" name="Register">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1647</width>
    <height>856</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Register</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(46, 47, 51);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>1000</width>
     <height>500</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>1850</width>
     <height>1920</height>
    </size>
   </property>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>-70</x>
      <y>0</y>
      <width>1691</width>
      <height>821</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QLineEdit" name="username">
      <property name="geometry">
       <rect>
        <x>640</x>
        <y>300</y>
        <width>401</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius: 30px;
background-color: rgb(255, 255, 255);
padding: 10px;
font: 700 10pt &quot;UD Digi Kyokasho NP-B&quot;;</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
      <property name="placeholderText">
       <string>Username</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="password">
      <property name="geometry">
       <rect>
        <x>640</x>
        <y>380</y>
        <width>401</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius: 30px;
background-color: rgb(255, 255, 255);
padding: 10px;
font: 700 10pt &quot;UD Digi Kyokasho NP-B&quot;;</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
      <property name="placeholderText">
       <string>Password</string>
      </property>
     </widget>
     <widget class="QLabel" name="Spotify_name">
      <property name="geometry">
       <rect>
        <x>820</x>
        <y>150</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 700 16pt &quot;UD Digi Kyokasho NP-B&quot;;
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Spotify</string>
      </property>
     </widget>
     <widget class="QLabel" name="spotify_logo">
      <property name="geometry">
       <rect>
        <x>750</x>
        <y>130</y>
        <width>61</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/new/prefix1/spotify logo.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="forgetPass">
      <property name="geometry">
       <rect>
        <x>730</x>
        <y>530</y>
        <width>221</width>
        <height>71</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 700 14pt &quot;UD Digi Kyokasho NP-B&quot;;
color: rgb(99, 100, 103);</string>
      </property>
      <property name="text">
       <string>Forgot Password</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="sign_button">
      <property name="geometry">
       <rect>
        <x>640</x>
        <y>460</y>
        <width>401</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 15pt &quot;Segoe UI Historic&quot;;
background-color: rgb(46, 189, 89);border-radius: 30px;
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>SIGN IN</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>690</x>
        <y>230</y>
        <width>301</width>
        <height>51</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="Sign_in">
         <property name="styleSheet">
          <string notr="true">border-radius:20px;
font: 15pt &quot;Segoe UI Historic&quot;;
color: rgb(255, 255, 255);
</string>
         </property>
         <property name="text">
          <string>SIGN IN</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="Sign_up">
         <property name="styleSheet">
          <string notr="true">border-radius:20px;
font: 15pt &quot;Segoe UI Historic&quot;;
color: rgb(255, 255, 255);
</string>
         </property>
         <property name="text">
          <string>SIGN UP</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QLineEdit" name="password_2">
      <property name="geometry">
       <rect>
        <x>640</x>
        <y>570</y>
        <width>401</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius: 30px;
background-color: rgb(255, 255, 255);
padding: 10px;
font: 700 10pt &quot;UD Digi Kyokasho NP-B&quot;;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
      <property name="placeholderText">
       <string>Password</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="username_2">
      <property name="geometry">
       <rect>
        <x>640</x>
        <y>500</y>
        <width>401</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius: 30px;
background-color: rgb(255, 255, 255);
padding: 10px;
font: 700 10pt &quot;UD Digi Kyokasho NP-B&quot;;</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
      <property name="placeholderText">
       <string>Username</string>
      </property>
     </widget>
     <widget class="QLabel" name="spotify_logo_2">
      <property name="geometry">
       <rect>
        <x>750</x>
        <y>130</y>
        <width>61</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/new/prefix1/spotify logo.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="Spotify_name_2">
      <property name="geometry">
       <rect>
        <x>820</x>
        <y>150</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 700 16pt &quot;UD Digi Kyokasho NP-B&quot;;
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Spotify</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget_2">
      <property name="geometry">
       <rect>
        <x>690</x>
        <y>230</y>
        <width>301</width>
        <height>51</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="Sign_in_2">
         <property name="styleSheet">
          <string notr="true">border-radius:20px;
font: 15pt &quot;Segoe UI Historic&quot;;
color: rgb(255, 255, 255);
</string>
         </property>
         <property name="text">
          <string>SIGN IN</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="Sign_up_2">
         <property name="styleSheet">
          <string notr="true">border-radius:20px;
font: 15pt &quot;Segoe UI Historic&quot;;
color: rgb(255, 255, 255);
</string>
         </property>
         <property name="text">
          <string>SIGN UP</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QLineEdit" name="username_3">
      <property name="geometry">
       <rect>
        <x>640</x>
        <y>360</y>
        <width>401</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius: 30px;
background-color: rgb(255, 255, 255);
padding: 10px;
font: 700 10pt &quot;UD Digi Kyokasho NP-B&quot;;</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
      <property name="placeholderText">
       <string>Date of Birth</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="username_4">
      <property name="geometry">
       <rect>
        <x>640</x>
        <y>290</y>
        <width>401</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius: 30px;
background-color: rgb(255, 255, 255);
padding: 10px;
font: 700 10pt &quot;UD Digi Kyokasho NP-B&quot;;</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
      <property name="placeholderText">
       <string>Email</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="username_5">
      <property name="geometry">
       <rect>
        <x>640</x>
        <y>430</y>
        <width>401</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius: 30px;
background-color: rgb(255, 255, 255);
padding: 10px;
font: 700 10pt &quot;UD Digi Kyokasho NP-B&quot;;</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
      <property name="placeholderText">
       <string>Country</string>
      </property>
     </widget>
     <widget class="QPushButton" name="sign_button_2">
      <property name="geometry">
       <rect>
        <x>640</x>
        <y>660</y>
        <width>401</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 15pt &quot;Segoe UI Historic&quot;;
background-color: rgb(46, 189, 89);border-radius: 30px;
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>SIGN UP</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_3"/>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1647</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
